### PR TITLE
Fix confusing setting label

### PR DESCRIPTION
**Pull request by aidalgol**
Remove "in web client" from "audible notifications" setting label, since Riot is
also a desktop application.

Signed-off-by: Aidan Gauland aidalgol+ghpr@fastmail.net

### CODE REVIEW

**Comment by t3chguy** [comment]
I think the catch there is it suggests that my phone will also not have audible notifications but I'm not sure it has that effect

**Comment by aidalgol** [comment]
So the iOS and Android Riot clients do not use this setting?

**Comment by t3chguy** [comment]
well they get notifications differently so they might not

**Comment by aidalgol** [comment]
Oh, right.  In either case, I think this is an improvement.

**Comment by lukebarnard1** [comment]
>Oh, right. In either case, I think this is an improvement.

I think I agree with @t3chguy here, it's ambiguous because the user might assume it affects all of their devices. If the point of this is to clarify the setting, then let's not leave it ambiguous.

How about "Enable audible web/desktop notifications"

**Comment by turt2live** [comment]
@aidalgol if we can get a sign off (as per the CONTRIBUTING.md) and the conflicts resolved, then we can merge this :)

For sign off, a comment on this PR with `Signed-off-by: Your Name <your_email@example.org>` will do.

File: src/i18n/strings/es.json [hunk 1]
```diff
@@ -714,7 +714,7 @@
     "Riot does not know how to join a room on this network": "Riot no sabe cómo unirse a una sala en esta red",
     "Set Password": "Establecer contraseña",
     "Enable audible notifications in web client": "Habilitar notificaciones audibles en el cliente web",
-    "Off": "Desactivado",
+    "Off": "Apagado",
```
Comment (turt2live) [inline src/i18n/strings/es.json]: this looks a bit accidental.

File: src/i18n/strings/lt.json [hunk 2]
```diff
@@ -109,7 +109,7 @@
     "Failed to set Direct Message status of room": "Nepavyko nustatyti tiesioginio pranešimo kambario būklės",
     "Monday": "Pirmadienis",
     "All messages (noisy)": "Visos žinutės (triukšmingas)",
-    "Enable them now": "Įjungti juos dabar",
+    "Enable them now": "Įgalinti juos dabar",
```
Comment (turt2live) [inline src/i18n/strings/lt.json]: also a bit accidental? I think these kinds of language fixes should be picked up by weblate come release time.

**Comment by jryans** [comment]
Thanks for making this contribution a while back. It looks more work was needed here to remove the other language file changes, but nothing happened after that. Would you like to move forward here?

**Comment by aidalgol** [comment]
@jryans What do I need to do?  Simply revert the changes to language files besides `en_EN.json`?

**Comment by jryans** [comment]
> @jryans What do I need to do? Simply revert the changes to language files besides `en_EN.json`?

Yes, that should do it!

**Comment by aidalgol** [comment]
I have reversed the unwanted changes to the language files, and remerged `develop` into the branch.

**Comment by jryans** [comment]
@aidalgol Hmm, I am still seeing many non-EN language files modified in the PR file view: https://github.com/matrix-org/matrix-react-sdk/pull/1995/files

**Comment by aidalgol** [comment]
@jryans I'm at a loss as to how to properly resolve this, then.

**Comment by jryans** [comment]
No worries! I have tried to carry the spirit of this change forward over in https://github.com/matrix-org/matrix-react-sdk/pull/2828. Thanks again for the suggestion!

**Pull request closed without merging**